applicationinsights-android/src/androidTest/java/com/microsoft/applicationinsights/TelemetryClientTest.java

```diff
@@ -91,13 +91,13 @@ public void testTrackException() throws Exception {
         LinkedHashMap<String, String> properties = new LinkedHashMap<>();
         LinkedHashMap<String, Double> measurements = new LinkedHashMap<>();
 
-        client.trackException(null);
-        client.trackException(new Exception());
+        client.trackHandledException(null);
+        client.trackHandledException(new Exception());
         try {
             throw new InvalidObjectException("this is expected");
         } catch (InvalidObjectException exception) {
-            client.trackException(exception);
-            client.trackException(exception, properties);
+            client.trackHandledException(exception);
+            client.trackHandledException(exception, properties);
```

Comment:
Just a note that we'll add a string parameter here at some point when we can directly send obfuscated Exception telemetry. Currently we send the Crash data type which is routed to the symbolication server. No change necessary now, just FYI.